content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,21 @@ menu:
 weight: 101
 ---
 
+## v2.0.9 [2021-09-23]
+
+### Features
+
+Add `--flux-log-enabled` option to [`influxd`](/influxdb/v2.0/reference/config-options/) to show detailed logs for Flux queries.
+
+### Bug fixes
+
+- Updating an inactive task no longer schedules it. Thanks @raffs!
+- Preserve comments in Flux queries when saving task definitions.
+- Resolve the compaction queue stats flutter.
+- Include `curl` as a dependency in `influxdb2` packages.
```

Comment:
does it make sense to move line 22 under features?

Reply:
This was more of a bug fix thing related to systemd so I think Bug Fixes is the right place for it...@codyshepherd might have thoughts as well.